warehouse/templates/includes/packaging/project-data.html

```diff
@@ -51,14 +71,12 @@ <h3 class="sidebar-section__title">{% trans %}Unverified details{% endtrans %}</
 {% if release.urls.values() | contains_valid_uris %}
```

Comment:
We need to update this as well, this should be:

```suggestion
{% if unverified_urls %}
```

We can lose the `contains_valid_uris` because it's already being done earlier.